util/utils.go

```diff
@@ -124,18 +124,18 @@ func IS2OP(x *big.Int, xLen int) []byte {
 	return x.FillBytes(buf)
 }
 
-const changeFactor = float64(1000000000)
+const coinUnit = float64(1000000000)
 
-// CoinToChange converts a coin amount to its corresponding change value.
-// Example: CoinToChange(2.75) returns 2750000000.
-func CoinToChange(coin float64) int64 {
-	return int64(coin * changeFactor)
+// ConvertCoinToCoinUnit converts a coin amount to its corresponding change value.
+// Example: ConvertCoinToCoinUnit(2.75) returns 2750000000.
+func ConvertCoinToCoinUnit(coin float64) int64 {
+	return int64(coin * coinUnit)
 }
 
-// ChangeToCoin converts a change value to its corresponding coin amount.
-// Example: ChangeToCoin(2750000000) returns 2.75.
-func ChangeToCoin(change int64) float64 {
-	return float64(change) / changeFactor
+// ConvertCoinUnitToCoin converts a change value to its corresponding coin amount.
```

Comment:
Please don't change this function name. Now we have many changes because of this rename.